### ISSUE TITLE
I was not offered the "push to batched" button when update reached days in testing

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#1875]
When https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2017-1e6d4edc57 reached 14 days in testing, a "push to stable" button appeared, but no "push to batched". Perhaps we only remembered to add "push to batched" for karma-based pushes?

I was able to request it for batched using the CLI, so there is a workaround:

```
$ bodhi updates request FEDORA-EPEL-2017-1e6d4edc57 batched
```

**Comment by bowlofeggs** [comment]
I thought this would be an easy fix, but the template code that does this is a mess unfortunately.

**Comment by bowlofeggs** [comment]
This is fixed in the 2.12.2 release.
